map/framework.cpp

```diff
@@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
 
 void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
```

Comment:
Тогда то, что ты мне сейчас рассказал, должно быть в документации к этому методу. Подробно объясняя, зачем нужен такой волшебный бубен, со ссылкой на mwmSet и поведение с дерегистрацией карт.

> On Mar 25, 2016, at 23:23, ygorshenin <notifications@github.com> wrote:
> 
> In map/framework.cpp <https://github.com/mapsme/omim/pull/2523#discussion_r57466254>:
> 
> > @@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
> >  
> >  void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
> Да, но только MwmSet-у все равно как ты его называешь ) Второй подход гибче, и позволяет как с удобством тестировать так и с удобством юзать.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2523/files/8bff1dbc61d252545ea4b21bffd137f33f43a0e2#r57466254>